backend/manager/modules/restapi/jaxrs/src/main/java/org/ovirt/engine/api/restapi/resource/BackendDiskSnapshotsResource.java

```diff
@@ -42,7 +42,8 @@ protected DiskSnapshots mapCollection(List<org.ovirt.engine.core.common.business
             collection.getDiskSnapshots().add(addLinks(populate(diskSnapshot, disk), Disk.class));
             diskSnapshot.setHref(buildHref(diskId.toString(), diskSnapshot.getId().toString()));
```

Comment:
Actually `toString()` is redundant on `diskSnapshot.getId()`. Already a `String`.